### PR TITLE
project_stage_closed: Add closed field to project.task class

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: project_stage_closed/models/__init__.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
+# Copyright 2018 Tecnativa - Ernesto Tejeda
```
Comment (astirpe) [inline project_stage_closed/models/__init__.py]: Please avoid to add the copyright line in the `__init__.py`, as stated in the guidelines

**Review by astirpe — CHANGES_REQUESTED** [review]
Thanks for this PR!

You should update the version in the manifest to `11.0.1.1.0`.

You may also want to add yourself in the contributors list.

**Comment by ernestotejeda** [comment]
Thank you @astirpe

**Review by astirpe — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
Thank you for the contribution. Why is this field needed?

**Comment by ernestotejeda** [comment]
This field allows us to know directly from another module if a task is closed without having to first ask if it has an associated stage and then ask if the stage is closed. For example, I can get all the tasks that are closed in this way: self.env ['project.task']. Search ([('closed', '=', True)])

**Comment by pedrobaeza** [comment]
@ernestotejeda This is not needed for searches. You can add this domain to the task search:

```python
self.env['project.task'].search([
    ...
    ('stage_id.closed', '=', False),
    ...
]
```

The only real need for this is to want to show something conditionally on task view, so we need a field in that model.

**Comment by pedrobaeza** [comment]
@ernestotejeda I guess this can be closed, isn't it?

**Comment by ernestotejeda** [comment]
Yes, I agree

**Pull request closed without merging**